doc/changelogs/CHANGELOG_V19.md

```diff
@@ -0,0 +1,209 @@
+  * [15.x](CHANGELOG_V15.md)
+  * [14.x](CHANGELOG_V14.md)
+  * [13.x](CHANGELOG_V13.md)
+  * [12.x](CHANGELOG_V12.md)
+  * [11.x](CHANGELOG_V11.md)
+  * [10.x](CHANGELOG_V10.md)
+  * [9.x](CHANGELOG_V9.md)
+  * [8.x](CHANGELOG_V8.md)
+  * [7.x](CHANGELOG_V7.md)
+  * [6.x](CHANGELOG_V6.md)
+  * [5.x](CHANGELOG_V5.md)
+  * [4.x](CHANGELOG_V4.md)
+  * [0.12.x](CHANGELOG_V012.md)
+  * [0.10.x](CHANGELOG_V010.md)
+  * [io.js](CHANGELOG_IOJS.md)
+  * [Archive](CHANGELOG_ARCHIVE.md)
+
+<a id="19.0.0"></a>
+
+## 2022-10-18, Version 19.0.0 (Current), @RafaelGSS and @ruyadorno
+
+Node.js 19 is here! Highlights include the update of the V8 JavaScript engine to 10.8, HTTP(s)/1.1 KeepAlive enabled by default, and ESM Resolution adjusts.
+
+Initially, Node.js 19 will replace Node.js 18 as our ‘Current’ release line. As per the release schedule, Node.js 18 ‘Hydrogen’ will be the 'LTS and Node.js 19 will be ‘Current' release for the next 6 months and then enter ‘Maintenance Mode’ in April 2023.
+
+### Notable Changes
+
+#### Deprecations and Removals
+
+* \[[`7dd2f41c73`](https://github.com/nodejs/node/commit/7dd2f41c73)] - **(SEMVER-MAJOR)** **module**: runtime deprecate exports double slash maps (Guy Bedford) [#44495](https://github.com/nodejs/node/pull/44495)
+* \[[`ada2d053ae`](https://github.com/nodejs/node/commit/ada2d053ae)] - **(SEMVER-MAJOR)** **process**: runtime deprecate coercion to integer in `process.exit()` (Daeyeon Jeong) [#44711](https://github.com/nodejs/node/pull/44711)
+
+#### HTTP(S)/1.1 KeepAlive by default
+
+Starting with this release, Node.js sets `keepAlive` to true by default. This means that any outgoing HTTP(s) connection will automatically use HTTP 1.1 Keep-Alive. The default waiting window is 5 seconds.
+Enable keep-alive will deliver better throughput as connections are reused by default.
+
+Additionally the agent is now able to parse the response `Keep-Alive` which the servers might send. This header instructs the client on how much to stay connected.
+On the other side, the Node.js HTTP server will now automatically disconnect idle clients (which are using HTTP Keep-Alive to reuse the connection) when `close()` is invoked).
+
+Node.js HTTP(S)/1.1 requests may experience a better throughput/performance by default.
+
+Contributed by Paolo Insogna in [#43522](https://github.com/nodejs/node/pull/43522)
+
+#### DTrace/SystemTap/ETW Support were removed
+
+The main reason is the lack of resources from Node.js team. The complexity to keep the support up-to-date has been proved not worth it without a clear plan to support those tools. Hence, [an issue was raised](https://github.com/nodejs/node/issues/44550) in the Node.js repository to assess a better support, for `DTrace` in specific.
+
+Contributed by Ben Noordhuis in [#43651](https://github.com/nodejs/node/pull/43651) and [#43652](https://github.com/nodejs/node/pull/43652)
+
+#### V8 10.7
+
+The V8 engine is updated to version 10.7, which is part of Chromium 106.
```

Comment:
Chromium 107? The first two parts of the V8 version is the Chromium version divided by 10.
Refs: https://v8.dev/docs/version-numbers